syz-manager/rpc.go

```diff
@@ -254,12 +270,20 @@ func (serv *RPCServer) shutdownInstance(name string) []byte {
 	runner.requests = nil
 	runner.mu.Unlock()
 
-	// If the object does not exist, there would be no oldRequests either.
-	fuzzerObj := serv.mgr.getFuzzer()
-	for _, req := range oldRequests {
+	if crashed {
 		// The VM likely crashed, so let's tell pkg/fuzzer to abort the affected jobs.
-		// TODO: distinguish between real VM crashes and regular VM restarts?
-		fuzzerObj.Done(req, &fuzzer.Result{Stop: true})
+		// fuzzerObj may be null, but in that case oldRequests would be empty as well.
+		fuzzerObj := serv.mgr.getFuzzer()
+		for _, req := range oldRequests {
+			fuzzerObj.Done(req, &fuzzer.Result{Stop: true})
+		}
+	} else {
+		// We will resend these inputs to another VM.
+		serv.mu.Lock()
+		for _, req := range oldRequests {
```

Comment:
serv.rescuedInputs = append(serv.rescuedInputs, oldRequests...)

Reply:
Will it really work? `oldRequests` is a map.

Reply:
Ah, ok, then it won't.